workshops/20_python_beginners_workshop/tasks/short_tasks/practice_strings&variables.md

```diff
@@ -0,0 +1,15 @@
+# Exercise 1: Strings & Variables
+
+1. Create a variable called `name` and assign your first name to that variable.
+2. Print `"Welcome to the Dungeon, xxx!"`. Use **%s** inside of your printing and put the variable `name` behind.
+
+**Note:** xxx has to be replaced by your name! Do not forget the space behind *Dungeon,*
```

Comment:
Python 3 tip:  There is a better way to do that with f-string. eg. 
```name = "Marie"
    print(f'Welcome to the Dungeon, {name}')
```
try to use that everywhere if you are in python 3

Reply:
we do not introduce this in the presentation, s. slide no. 10.